airmozilla/popcorn/static/popcorn/js/editor.js

```diff
@@ -1,2 +1,27 @@
+function getSlug() {
+  var pathname = window.location.pathname,
+      re = /\/popcorn\/edit\/(.+)\//,
+      match = pathname.match(re);
+
+  return match[1];
+}
+
+PopcornEditor.listen(PopcornEditor.events.loaded, function () {
+  $.ajax({
+    method: 'GET',
+    url: '/popcorn/data/',
```

Comment:
We need to eradicate all hardcoded URLs from the static files. 
This should only exist has a written string in one single place `popcorn/urls.py`.
The way to "transfer" such a value from django into a script is to use a tag and a `data` attribute.  

Here's one example: https://github.com/mozilla/airmozilla/blob/master/airmozilla/main/templates/main/main_base.html#L169-L175
https://github.com/mozilla/airmozilla/blob/master/airmozilla/main/static/main/js/autocompeter.js#L7-L11

You don't have to use a non-standard tag. Perhaps you can attach...
```
data-editor-url="{{ url('popcorn:data') }}"
```
... on the `body` tag or something.